lib/internal/socket_list.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
```

Comment:
Strange, keep getting a lint error when using this. https://gist.github.com/evanlucas/0920f2b5cd90340cca27

Any ideas @silverwind? I've tried adding in an `object-shorthand` rule, but it didn't seem to do anything